### PR TITLE
Added bank-of-anthos predefined workflow

### PR DESCRIPTION
**Pull request by oumkale**
Signed-off-by: Oum Kale <oumkale@chaosnative.com>
 
 - Added bank-of-anthos predefined workflow
 - Consist Http probe against frontend
 - Network loss is done on a balancereader/transactionhistory pod which is Deployment and runs as a single replica in 
   weak scenario and double replica in resilient state
 - Due to loss in pod is inaccessible (or Read-Only) leading to failed requests

### CODE REVIEW

File: workflows/bank-of-anthos/bank-of-anthos.chartserviceversion.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    It install bank-of-anthos application, generate loads, inject chaos on bank-of-anthos,
+    uninstall the application and revert the chaos
+  experiments:
+    - pod-network-loss
+  keywords:
+    - Kubernetes
+    - bank-of-anthos
+    - Pod
+    - Network
+    - Loss
```
Comment (ispeakc0de) [inline workflows/bank-of-anthos/bank-of-anthos.chartserviceversion.yaml]:
```suggestion
```

File: workflows/bank-of-anthos/bank-of-anthos.chartserviceversion.yaml [hunk 2]
```diff
@@ -0,0 +1,40 @@
+apiVersion: litmuchaos.io/v1alpha1
+kind: ChartServiceVersion
+metadata:
+  name: bank-of-anthos
+  version: 0.1.0
+  annotations:
+    categories: bank-of-anthos
+    chartDescription: Injects chaos on bank-of-anthos application using cmd probe.
```
Comment (ispeakc0de) [inline workflows/bank-of-anthos/bank-of-anthos.chartserviceversion.yaml]:
```suggestion
    categories: bank-of-anthos
    chartDescription: Injects chaos on bank-of-anthos application using http probe.
```

File: workflows/icons/anthos.svg [hunk 3]
```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Generator: Adobe Illustrator 24.2.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
+<svg version="1.0" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
+	 viewBox="0 0 300 280" enable-background="new 0 0 300 280" xml:space="preserve">
```
Comment (ispeakc0de) [inline workflows/icons/anthos.svg]: can you check this?

Comment (ispeakc0de) [reply workflows/icons/anthos.svg]: also, rename the icon name to `bank-of-athos`

Comment (oumkale) [reply workflows/icons/anthos.svg]: This one is .svg format we have given link at 
in bank-of-anthos.chartserviceversion.yaml
    - url: https://raw.githubusercontent.com/litmuschaos/chaos-charts/master/workflows/icons/anthos.svg

**Review by ispeakc0de — APPROVED** [review]

**Comment by ksatchit** [comment]
In updated versions of this workflow we might want to carry out banking transaction via probe using POST in addition to checking frontend availability (the latter will be true even in cases of balancereader inaccessibility, for example)

**Review by ksatchit — APPROVED** [review]

**Comment by oumkale** [comment]
Yes @ksatchit, 
We will add POST http probe for banking transaction soon.

**Pull request merged**